src/demoserver.cpp

```diff
@@ -220,14 +220,20 @@ void DemoServer::handle_packet(AOPacket packet)
               client_sock->write(packet.toUtf8());
           }
         }
+        else if (contents[1].startsWith("/reload"))
+        {
+            load_demo(p_path);
+            QString packet = "CT#DEMO#" + tr("Current demo file reloaded. Send /play or > in OOC to begin playback.") + "#1#%";
+            client_sock->write(packet.toUtf8());
```

Comment:
You may want to reset the playback timer. There are also stateful components like the countdown timer you might need to reset/hide.

Reply:
Ohhh good catch